extensions/integration-tests/package.json

```diff
@@ -45,19 +44,20 @@
     ]
   },
   "scripts": {
+    "fmktest": "mocha -r ts-node/register --watch-extensions tsx,ts -u tdd test/**/*.test.ts",
     "vscode:prepublish": "node ../../node_modules/gulp/bin/gulp.js --gulpfile ../../build/gulpfile.extensions.js compile-extension:vscode-colorize-tests ./tsconfig.json",
     "postinstall": "node ./node_modules/vscode/bin/install"
   },
   "devDependencies": {
-    "@types/node": "7.0.43",
-    "@types/chai": "3.4.34",
+    "@types/node": "^10.12.12",
+    "@types/chai": "^4.1.7",
     "mocha-junit-reporter": "^1.17.0",
     "mocha-multi-reporters": "^1.1.7",
-    "vscode": "1.1.5",
-    "chai": "3.5.0"
+    "vscode": "^1.1.5",
```

Comment:
you can always directly refer to the .d.ts in the src folder rather than pulling it in as a dependency

Reply:
Agreed - we need to clean up here, but please note that I did not add this dependency so will defer the cleanup necessary for future checkin.